src/peerstate.rs

```diff
@@ -271,14 +273,36 @@ impl Peerstate {
                 .query_get_value("SELECT id FROM contacts WHERE addr=?;", paramsv![self.addr])
                 .await?
             {
-                let chat_id = ChatIdBlocked::get_for_contact(context, contact_id, Blocked::Request)
-                    .await?
-                    .id;
-
+                let chats = Chatlist::try_load(context, 0, None, contact_id)
+                    .await
+                    .unwrap();
```

Comment:
```suggestion
                    .await?;
```

a single `unwrap()` will just crash the app. `?` is a shortcut for "in case of errors, return the error result to the caller"

Reply:
thanks for the tip, I had no idea what those `?` meant :D